src/NuGet.Core/NuGet.Packaging/Signing/Signatures/SigningCertificate.cs

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using NuGet.Packaging.Signing.DerEncoding;
+
+namespace NuGet.Packaging.Signing
+{
+    /*
+        From RFC 2634 (https://tools.ietf.org/html/rfc2634#section-5.4):
+
+            SigningCertificate ::= SEQUENCE {
+                certs        SEQUENCE OF ESSCertID,
+                policies     SEQUENCE OF PolicyInformation OPTIONAL
+            }
+    */
+    /// <remarks>This is public only to facilitate testing.</remarks>
+    public sealed class SigningCertificate
+    {
+        public IReadOnlyList<EssCertId> Certificates { get; }
+
+        private SigningCertificate(IReadOnlyList<EssCertId> certificates)
+        {
+            Certificates = certificates;
+        }
+
+        public static SigningCertificate Read(byte[] bytes)
+        {
+            var reader = new DerSequenceReader(bytes);
+
+            return Read(reader);
+        }
+
+        internal static SigningCertificate Read(DerSequenceReader reader)
+        {
+            var essCertIdReader = reader.ReadSequence();
+            var certificates = ReadCertificates(essCertIdReader);
+
+            // Skip the "policies" field.  We do not use it.
```

Comment:
You should strive to make sure the data is as conforming as possible, so that you don't have to worry about maintaining compatibility with loosey-goosey parsing later, should you switch to a more strict reader.